app/models/discourse_post_event/event_date.rb

```diff
@@ -41,11 +41,6 @@ def upsert_topic_custom_field
       end
     end
 
-    after_commit :reset_invitees_status, on: %i[create]
```

Comment:
the only place where we call event_dates.create and trigger this hook, is in the event hook and we manually call the set_date which has the same behavior we just have modified in this commit. It was essentially doing two times the same work.